Detectors/CTP/workflowLumi/src/RawToLumiConverterSpec.cxx

```diff
@@ -0,0 +1,154 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <string>
+#include "FairLogger.h"
+#include "CommonDataFormat/InteractionRecord.h"
+#include "Framework/InputRecordWalker.h"
+#include "Framework/DataRefUtils.h"
+#include "Framework/WorkflowSpec.h"
+#include "DetectorsRaw/RDHUtils.h"
+#include "DPLUtils/DPLRawParser.h"
+#include "CTPWorkflowLumi/RawToLumiConverterSpec.h"
+#include "CTPWorkflow/RawToDigitConverterSpec.h"
+#include "CommonUtils/VerbosityConfig.h"
+
+using namespace o2::ctp::lumi_workflow;
+
+void RawToLumiConverterSpec::init(framework::InitContext& ctx)
+{
+}
+
+void RawToLumiConverterSpec::run(framework::ProcessingContext& ctx)
+{
+  mOutputLumiPoints.clear();
+  const gbtword80_t bcidmask = 0xfff;
+  using InputSpec = o2::framework::InputSpec;
+  using ConcreteDataTypeMatcher = o2::framework::ConcreteDataTypeMatcher;
+  using Lifetime = o2::framework::Lifetime;
+  // mOutputHWErrors.clear();
+  std::vector<InputSpec> filter{InputSpec{"filter", ConcreteDataTypeMatcher{"CTP", "RAWDATA"}, Lifetime::Timeframe}};
```

Comment:
you don't need a filter here since you the workflow subscribes to CTP/RAWDATA only.

Reply:
I am not sure if I need to keep filter.

Reply:
You don't need it.